osu.Game/Overlays/Dashboard/Dashboard/DrawableBeatmapsList.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using System.Collections.Generic;
+using System.Linq;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Game.Beatmaps;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osuTK;
+
+namespace osu.Game.Overlays.Dashboard.Dashboard
+{
+    public abstract class DrawableBeatmapsList : CompositeDrawable
+    {
+        private readonly List<BeatmapSetInfo> beatmaps;
+
+        protected DrawableBeatmapsList(List<BeatmapSetInfo> beatmaps)
+        {
+            this.beatmaps = beatmaps;
+        }
+
+        [BackgroundDependencyLoader]
+        private void load(OverlayColourProvider colourProvider)
+        {
+            FillFlowContainer flow;
+
+            RelativeSizeAxes = Axes.X;
+            AutoSizeAxes = Axes.Y;
+            InternalChild = flow = new FillFlowContainer
+            {
+                RelativeSizeAxes = Axes.X,
+                AutoSizeAxes = Axes.Y,
+                Direction = FillDirection.Vertical,
+                Spacing = new Vector2(0, 10),
+                Children = new Drawable[]
+                {
+                    new OsuSpriteText
+                    {
+                        Font = OsuFont.GetFont(size: 16, weight: FontWeight.Bold),
+                        Colour = colourProvider.Light1,
+                        Text = CreateTitle(),
+                        Padding = new MarginPadding { Left = 10 }
+                    }
+                }
+            };
+
+            flow.AddRange(beatmaps.Select(CreateBeatmapPanel));
+        }
+
+        protected abstract string CreateTitle();
```

Comment:
```suggestion
        protected abstract string Title { get; }
```

the ballpark rule is: if it's simple data, there's no reason to have it as a method; methods generally *do* or construct stuff rather than just return stuff